component/prometheus.jsonnet

```diff
@@ -0,0 +1,189 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.rancher_monitoring;
+local namespace = params.namespace;
+
+local instance = params.prometheusInstance;
+local name = 'prometheus-' + instance;
+
+local labels = {
+  'app.kubernetes.io/name': 'prometheus',
+  'app.kubernetes.io/instance': instance,
+  'app.kubernetes.io/managed-by': 'syn',
+};
+
+local matchLabels = {
+  prometheus: instance,
+  role: 'alert-rules',
+};
+
+local nsSelector = {
+  matchLabels: {
+    SYNMonitoring: 'main',
+  },
+};
+
+// The actual prometheus resource for the prometheus operator
+local prometheus = {
```

Comment:
Again, we might be able to profit from `kube-prometheus`.

Reply:
see comment above